routes/api.js

```diff
@@ -52,6 +52,21 @@ router.route("/collections/:caffeine").get((req, res) => {
 router.route("/products/:collection/:tag/:caffeinated").get((req, res) => {
   let calledRoute = `/api/products/${req.params.collection}/${req.params.tag}/${req.params.caffeinated}`;
   console.log(`${calledRoute} called`);
+
+  // create new db entry w/ req params
+
+  db.Stats.create({
+    caffeinated: req.params.caffeinated == 'caffeine',
+    collection_handle: req.params.collection,
+    bucket: req.params.tag
+  })
+  .then(data => {
+    console.log("logged stats: " + data);
+  })
+  .catch(err => {
+    console.error(err);
+  });
```

Comment:
This route function is getting a bit long. Could you refactor this code into a separate function (e.g. logSelectionStats(caffeine, collection, bucket) and call that here instead?